app/controllers/boxes_controller.rb

```diff
@@ -100,12 +113,37 @@ def load_box
     end
   end
 
+  def load_box_print
+
+    @box = Box.where(institution: @navigation_context.institution, id: params.fetch(:id)).take
+    if @boxes.nil?
+      @box = Box
```

Comment:
> Where is `@boxes` defined? Did you mean `@box`? In which case:
> 
> ```ruby
> @box ||= Box.joins(...).where(...)
> ```

😅 yes, it's @box, I think it happened because I copied from a comment, talking to Johannes.